JpegXmpWritePluginMDE/MetadataExtractor/Formats/Jpeg/JpegFragment.cs

```diff
@@ -44,7 +38,7 @@ public JpegFragment([NotNull] byte[] bytes, [CanBeNull] JpegSegmentPlugin segmen
 		public override string ToString()
 		{
 			if (IsSegment)
-				return $"{Bytes.Length} bytes ({Segment.Type})";
+				return $"{Bytes.Length} bytes ({Segment!.Type})";
```

Comment:
May or may not be possible to avoid the ! by annotating IsSegment, I can't recall offhand